packages/adapter-node/src/server.js

```diff
@@ -29,8 +29,11 @@ export function createServer({ render }) {
 
 	const assets_handler = fs.existsSync(paths.assets)
 		? sirv(paths.assets, {
-				maxAge: 31536000,
-				immutable: true
+				setHeaders: (res, pathname, stats) => {
+					if (pathname.startsWith('/_app/')) {
```

Comment:
I think this should use the configured value instead of it being hardcoded. See Rich's earlier comment where `appDir` is provided by the main configuration: https://kit.svelte.dev/docs#configuration-appdir

Reply:
@benmccann  Is there a variable I can use instead?

Reply:
It looks like this is basically the request made in https://github.com/sveltejs/kit/issues/1435. Perhaps @Rich-Harris has thoughts on how to accomplish this?

Reply:
There's a PR for this now: https://github.com/sveltejs/kit/pull/1559

Reply:
@bjon https://github.com/sveltejs/kit/pull/1559 has been merged. If you rebase this PR hopefully you can you the new variable to address this

Reply:
@benmccann I'm not sure how I should access the config object in `adapt`, from `src/server.js`?
Should I store it to disk, and read that from `src/server.js`?

Reply:
Hmm. Can you use esbuild defines? https://esbuild.github.io/api/#define

Reply:
Never used esbuild, but I can look into that.

Reply:
If I'm not mistaken, adapter-node isn't using esbuild at all

Reply:
Ok, now I think I get it. `adapter-node` builds `files/server.js` with Rollup. Then `adapt()` copies it to `/build`. I don't really have a good solution for this, with the current build system.

Reply:
I think that @Conduitry was looking into converting it to use esbuild

Reply:
I was, but I sort of ran into a wall because I don't know how to use esbuild. In particular, I was having trouble configuring which dependencies get bundled and which remain external dependencies. So if other folks are thinking about taking a look at this, it would be appreciated. Don't worry about duplicated effort, because I'm not on the verge of opening a PR for that or anything.

Reply:
The PR for esbuild has been opened at https://github.com/sveltejs/kit/pull/1648

Reply:
@benmccann this should be fixed now